tests/test_auto_detection.py

```diff
@@ -134,64 +140,58 @@ def test_knossos_dataset_name_and_layer_path_detection() -> None:
         # test if dataset name and layer name and mag are correct
         converter = KnossosConverter()
         converter.source_files = [
-            prefix
-            + "knossos/color/1/x0000/y0000/z0000/test_mag1_x0000_y0000_z0000.raw",
-            prefix
-            + "knossos/color/1/x0000/y0000/z0001/test_mag1_x0000_y0000_z0001.raw",
-            prefix
-            + "knossos/color/1/x0000/y0001/z0000/test_mag1_x0000_y0001_z0000.raw",
+            f"{prefix}knossos{sep}color{sep}1{sep}x0000{sep}y0000{sep}z0000{sep}test_mag1_x0000_y0000_z0000.raw",
```

Comment:
I find the many `{sep}`s quite noisy. Maybe we should implement a small helper, like this?

```suggestion
            adapt_separators(f"{prefix}knossos/color/1/x0000/y0000/z0000/test_mag1_x0000_y0000_z0000.raw"),
```

and

```
def adapt_separators(s):
  return s.replace('/', os.sep)
```

But I can understand if you want to stick to the current solution.